From: Antonio Terceiro <terceiro@abel.debian.org>
Date: Mon, 19 Apr 2021 17:50:12 +0000
Subject: Revert "Make host_* values consistent with target_*"

See https://bugs.ruby-lang.org/issues/17021 for the original request.

This breaks the build on the 32-bit Debian ARM architectures. Maybe this
is specific to the Debian packaging, so this has not been reported
upstream yet.
---
 configure.ac | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index da22ab6..73ce534 100644
--- a/configure.ac
+++ b/configure.ac
@@ -309,14 +309,9 @@ AC_SUBST(CC_VERSION_MESSAGE, $cc_version_message)
 : ${DLDFLAGS="$LDFLAGS"}
 
 RUBY_UNIVERSAL_ARCH
-AS_IF([test "$target_cpu" != "$host_cpu" -a "$GCC" = yes -a "$cross_compiling" = no -a "${universal_binary:-no}" = no], [
+AS_IF([test "$target_cpu" != "$host_cpu" -a "$GCC" = yes -a "$cross_compiling" = no -a "$universal_binary" = no], [
     RUBY_DEFAULT_ARCH("$target_cpu")
 ])
-host_os=$target_os
-host_vendor=$target_vendor
-host_cpu=$target_cpu
-host=$target
-host_alias=$target_alias
 
 AS_CASE(["$target_os"], [darwin*], [
 if libtool 2>&1 | grep no_warning_for_no_symbols > /dev/null; then
